zz.home-work/src/components/__tests__/todo-form.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { TodoForm } from '../todo-form';
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+
+describe('TodoForm', () => {
+  let mockSetTodos: ReturnType<typeof vi.fn>;
+  let todoInput: HTMLElement;
+  let dateInput: HTMLElement;
+  let addButton: HTMLElement;
+
+  beforeEach(() => {
+    mockSetTodos = vi.fn();
+    render(<TodoForm todos={[]} setTodos={mockSetTodos} />);
+
+    todoInput = screen.getByLabelText('New Todo');
+    dateInput = screen.getByLabelText('Deadline');
+    addButton = screen.getByRole('button', { name: 'Add Todo' });
+
+    vi.clearAllMocks();
+  });
```

Comment:
```suggestion
  beforeEach(() => {
    vi.clearAllMocks();
    mockSetTodos = vi.fn();
    render(<TodoForm todos={[]} setTodos={mockSetTodos} />);

    todoInput = screen.getByLabelText('New Todo');
    dateInput = screen.getByLabelText('Deadline');
    addButton = screen.getByRole('button', { name: 'Add Todo' });
  });
```

상단에 위치시켜서 이전에 모킹했던 내용을 먼저 클리어하는 것이 좋을 듯 합니다 ~